sparse_strips/vello_hybrid/shaders/sparse_strip_renderer.wgsl

```diff
@@ -101,11 +104,9 @@ fn fs_main(in: VertexOutput) -> @location(0) vec4<f32> {
         let texel_index = alphas_index / 4u;
         // Which channel (R,G,B,A) in the texel
         let channel_index = alphas_index % 4u;
-        // Calculate texture coordinates using bitwise operations
-        // This is more efficient than using modulo and division when width is a power of 2
-        let alphas_tex_width_bits = firstTrailingBit(alphas_tex_width);
```

Comment:
WebGL2 doesn't support `firstTrailingBit` -> Need to precompute

Reply:
What's the failure mode here? Is it an error from wgpu, or does the webgl implementation give an error? If the webgl implementation gives an error, I think the wgpu team would like to know this.

If wgpu is erroring, ideally, we would tell wgpu to run in a fully webgl2 compatible mode, cross-platform. But that's not to be dealt with in this PR.

Reply:
> If wgpu is erroring, ideally, we would tell wgpu to run in a fully webgl2 compatible mode, cross-platform. But that's not to be dealt with in this PR.

WGPU is erroring - Yes, totally agree!

Reply:
I *think* that's not a wgpu error. That is, it's wgpu just forwarding the failure generated by your browser. It looks very similar to https://github.com/gfx-rs/wgpu/issues/4525

Reply:
Ah yes - you're right!